spring-cloud-commons/src/main/java/org/springframework/cloud/client/serviceregistry/Registration.java

```diff
@@ -0,0 +1,7 @@
+package org.springframework.cloud.client.serviceregistry;
+
+/**
+ * @author Spencer Gibb
+ */
+public interface Registration {
```

Comment:
Yeah, leftover from a refactor where this interface had a method and I removed it.